lib/api/model/model.dart

```diff
@@ -382,6 +390,51 @@ enum ChannelPostPolicy {
   final int? apiValue;
 
   int? toJson() => apiValue;
+
+  static ChannelPostPolicy fromApiValue(int value) => _byApiValue[value]!;
```

Comment:
… Hmm, there's actually a shortage of enums that work quite like this. (Where it's a numeric value, rather than a string that's the enum value's name, and where at the same time we have a named method we call to deserialize it rather than always doing so as a field on a `JsonSerializable` class.)

Our enum handling in general feels a bit messy — and I think that's largely because `json_serializable` is a bit messy in its API for enums — but this particular one is fine for now. Later sometime we can try to find a nicer pattern and sweep through cleaning them up.